test/builtin/test_compile.py

```diff
@@ -44,20 +47,38 @@
     )
 
 
-def test_pythonize_code():
+def test_compile_code():
     for str_expr, x, res in [
         ("Sin[x]", 1.5, 0.997495),
         ("Exp[-x^2/2.]", 0.0, 1.0),
         ("Sqrt[x]", 1.0, 1.0),
+        ("BesselJ[0,x]", 0.0, 1.0),
+        ("Exp[BesselJ[0,x]-1.]", 0.0, 1.0),
     ]:
 
         expr = session.evaluate("Compile[{x}, " + str_expr + " ]")
         assert expr.get_head_name() == "System`CompiledFunction"
         assert len(expr.elements) == 3
         code = expr.elements[2]
         assert isinstance(code, CompiledCode)
-        print(code.cfunc)
         y = code.cfunc(x)
+        print(str_expr, code.cfunc)
+        assert abs(y - res) < 1.0e-6
+
+
+def test_pythonize_code_1():
```

Comment:
A very small thing, but what this test seems to be doing is testing builtin functions with symbols in them. The logic by which this is about pythonizing (which isn't really a word) is at best confusing and jarring jargon. 

A suggestion for an alternate name might be `test_builtin_fns_with_symbols`

Reply:
Sure.